### ISSUE TITLE
Can not delete the notebook

### ISSUE THREAD

**Issue by huhuhang** [jupyterhub/jupyterhub#1760]
Hi! Thanks for using JupyterHub.

If you are reporting an issue with JupyterHub:

- Please use the [GitHub issue](https://github.com/jupyterhub/jupyterhub/issues)
  search feature to check if your issue has been asked already. If it has,
  please add your comments to the existing issue.

- Where applicable, please fill out the details below to help us troubleshoot
  the issue that you are facing. Please be as thorough as you are able to
  provide details on the issue.

**How to reproduce the issue**
delete a notebook.

**What you expected to happen**
send it to trash.

**What actually happens**
did not work.

**Share what version of JupyterHub you are using**
0.8.1

Running `jupyter troubleshoot` from the command line, if possible, and posting
its output would also be helpful.

```

Insert jupyter troubleshoot output here

Traceback (most recent call last): File "/opt/conda/lib/python3.6/site-packages/tornado/web.py", line 1512, in _execute result = yield result File "/opt/conda/lib/python3.6/site-packages/tornado/gen.py", line 1055, in run value = future.result() File "/opt/conda/lib/python3.6/site-packages/tornado/concurrent.py", line 238, in result raise_exc_info(self._exc_info) File "<string>", line 4, in raise_exc_info File "/opt/conda/lib/python3.6/site-packages/tornado/gen.py", line 307, in wrapper yielded = next(result) File "/opt/conda/lib/python3.6/site-packages/notebook/services/contents/handlers.py", line 235, in delete yield gen.maybe_future(cm.delete(path)) File "/opt/conda/lib/python3.6/site-packages/notebook/services/contents/manager.py", line 278, in delete self.delete_file(path) File "/opt/conda/lib/python3.6/site-packages/notebook/services/contents/filemanager.py", line 499, in delete_file send2trash(os_path) File "/opt/conda/lib/python3.6/site-packages/send2trash/plat_other.py", line 191, in send2trash dest_trash = find_ext_volume_trash(topdir) File "/opt/conda/lib/python3.6/site-packages/send2trash/plat_other.py", line 151, in find_ext_volume_trash trash_dir = find_ext_volume_fallback_trash(volume_root) File "/opt/conda/lib/python3.6/site-packages/send2trash/plat_other.py", line 141, in find_ext_volume_fallback_trash check_create(trash_dir) File "/opt/conda/lib/python3.6/site-packages/send2trash/plat_other.py", line 86, in check_create os.makedirs(dir, 0o700) File "/opt/conda/lib/python3.6/os.py", line 220, in makedirs mkdir(name, mode) NotADirectoryError: [Errno 20] Not a directory: b'/home/jovyan/Untitled.ipynb/.Trash-1000'


```

![wx20180329-093044](https://user-images.githubusercontent.com/5147530/38065104-0708327c-3334-11e8-915c-5c7580986b38.png)
![wx20180329-093218](https://user-images.githubusercontent.com/5147530/38065124-175ea764-3334-11e8-8719-c34951ffb1f7.png)

**Comment by willingc** [comment]
Hi @huhuhang,

Which version of tornado and notebook are you using?

**Comment by betatim** [comment]
I think this should be fixed by https://github.com/jupyterhub/jupyterhub/pull/1688.

**Comment by betatim** [comment]
Closing this because it has been fixed in #1688.
